src/main/scala/datalog/execution/StagedCompiler.scala

```diff
@@ -175,9 +178,8 @@ class StagedCompiler(val storageManager: StorageManager)(using val jitOptions: J
               '{ $acc ; def eval_sn_lambda() = $next; eval_sn_lambda() }
             )
           case _ =>
-            cOps.reduceLeft((acc, next) => // TODO[future]: make a block w reflection instead of reduceLeft for efficiency
-              '{ $acc ; $next }
-            )
+            // TODO[future]: make a block w reflection instead of reduceLeft for efficiency
```

Comment:
```suggestion
            // TODO[future]: make a block w reflection instead of reduceLeft for efficiency
```
```suggestion
            // TODO[future]: make a block w reflection instead of reduceLeft for efficiency
```